kubernetes/workflows.yaml

```diff
@@ -114,6 +114,13 @@ workflows:
 
             Also supported are :code:`env` and :code:`volumes` for defining the environment variables and
             volumes specific to this step.
+        - name: generateName
```

Comment:
should this be `somethingPrefix` like `jobNamePrefix` vs. `generateName` (which I'd have guessed would be a bool based on the parameter name)?

Reply:
`generateName` matches the kubernetes job schema. I can change it if you want.